ouroboros-consensus-cardano-tools/src/Cardano/Tools/DBAnalyser/Run.hs

```diff
@@ -118,7 +119,10 @@ analyse DBAnalyserConfig{analysis, confLimit, dbDir, selectDB, validation, verbo
             putStrLn $ "ImmutableDB tip: " ++ show tipPoint
             pure result
         SelectChainDB -> do
-          bs <- newBackingStore (newBackingStoreInitialiser nullTracer LedgerDB.InMemoryBackingStore) ledgerDbFS (projectLedgerTables genesisLedger)
+          Debug.traceShowM "before"
+          let !x = projectLedgerTables genesisLedger
+          Debug.traceShowM "after"
```

Comment:
I've been able to narrow down where the `ouroboros-consensus-cardano-tools` test are hanging to this part here. `before` is traced, while `after` is not